src/hooks/useGeneralTlrSettings.js

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from 'react-query';
```

Comment:
Please add `react-query v3` to dependencies

Reply:
Fixed